lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -204,6 +204,28 @@ describe('BasicLtiLaunchApp', () => {
       );
     });
 
+    it('does not create a second auth window when clicked twice', async () => {
+      // Make the initial URL fetch request reject with an unspecified `ApiError`.
+      fakeApiCall.rejects(new ApiError(400, {}));
+
+      const wrapper = renderLtiLaunchApp();
+      const authButton = await waitForElement(
+        wrapper,
+        'Button[label="Authorize"]'
+      );
+      await contentHidden(wrapper);
```

Comment:
Is this needed? It seems that the rest of the test only needs to verify that the `authButton` element is shown.

Reply:
Its redundant along with the the previous test. Can be removed in both places.